biomaj/schema_version.py

```diff
@@ -104,3 +107,74 @@ def migrate_pendings():
             print("Migration: %d bank production info updated" % updated)
 
         schema.update_one({'id': 1}, {'$set': {'version': installed_version}})
```

Comment:
why do you update schema, schema is not update, it is used for schema modification.